code/game/machinery/firealarm.dm

```diff
@@ -10,7 +10,7 @@
 	result_path = /obj/machinery/firealarm
 
 /obj/machinery/firealarm
-	name = "fire alarm"
+	name = "fire/flood alarm"

```

Comment:
Air Safety Alarm instead?
Its really just using the same system and makes more sense to keep things merged down to prevent conflicts